package.json

```diff
@@ -54,31 +54,73 @@
     "@types/commander": "^2.12.2",
     "@types/configstore": "^4.0.0",
     "async-mutex": "^0.2.4",
+    "axios": "^0.19.0",
+    "body-parser": "^1.19.0",
+    "bunyan": "^1.8.14",
+    "chai": "^4.2.0",
+    "chai-as-promised": "^7.1.1",
     "chalk": "^4.0.0",
     "commander": "^5.1.0",
     "configstore": "^5.0.1",
-    "connect": "^3.7.0",
+    "connect-ensure-login": "^0.1.1",
+    "cookie-parser": "^1.4.5",
+    "cors": "^2.8.5",
```

Comment:
Because this is not a web app, we can just set our access control to be `*` for everything. Therefore a specialized `cors` library should not be necessary. Check the MDN docs for this.

Reply:
same here, this will be removed